### ISSUE TITLE
Have code coverage of sormas-app in SonarQube

### ISSUE THREAD

**Issue by StefanKock** [hzi-braunschweig/SORMAS-Project#1736]
### Situation Description
Currently the source code of _sormas-app_ gets analysed and issues pop up, but unit tests and code coverage is missing. This was not included in #270.

### Feature Description
Have unit tests and code coverage of _sormas-app_ in SonarQube.

### Possible Alternatives
Do nothing.

### Additional Information
Keep the analysis separate from _sormas-base_, see #1729.

**Comment by StefanKock** [comment]
Ways to integrate:

- https://docs.gradle.org/current/userguide/jacoco_plugin.html
- https://medium.com/@vialyx/android-code-coverage-metrics-with-sonarqube-jacoco-3f99bc700e84
- https://www.rallyhealth.com/coding/code-coverage-for-android-testing

**Comment by StefanKock** [comment]
I managed to append the gradle configuration of the sonarqube plugin and combine it with a jacoco report.
The setup is combined of these two tutorials, but only for unit tests and not the androidTests since cannot yet run them on Jenkins.
- https://www.rallyhealth.com/coding/code-coverage-for-android-testing
- https://tech.thalia.de/sonarqube-integration-in-einem-android-projekt/

New gradle task to call: `:app:jacocoUnitTestReport` (before `:app:sonarqube`).

I tested that Code Coverage is recognized in `SonarQube/sormas-app`, but not in `sormas-base`. Unfortunatly, the xml analysis is now missing and the unit tests are still not linked in SonarQube.
